doc/orchestrator.rst

```diff
@@ -165,3 +165,541 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+  colo_client.log_data(LLInfo, f"The colocated db has tensor_2: {colo_val2}")
+
+The output will be as follows::
+
+  Model: colocated logger@00-00-00:The colocated db has tensor_1: True
+  Model: colocated logger@00-00-00:The colocated db has tensor_2: True
+
+The Experiment Driver Script
+----------------------------
+To run the previous application, we must define workflow stages within a workload.
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime.
+In this example, we instantiate an ``Experiment`` object with the name ``getting-started-multidb``.
+We setup the SmartSim ``logger`` to output information from the Experiment.
+
+.. code-block:: python
+
+  from smartsim import Experiment
+  from smartredis import Client
+  from smartredis.log import get_logger
+  import numpy as np
+
+  exp = Experiment("getting-started-multidb", launcher="auto")
+  logger = get_logger("Multidb Experiment Log")
+
+Launch Multiple Orchestrators
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the context of this ``Experiment``, it's essential to create and launch
+the databases as a preliminary step before any other components since
+the application script requests tensors from the launched databases.
+
+We aim to showcase the multi-database automation capabilities of SmartSim, so we
+create two databases in the workflow: a single-sharded database and a
+multi-sharded database.
+
+Step 1: Initialize Orchestrators
+""""""""""""""""""""""""""""""""
+To create an database, utilize the ``Experiment.create_database()`` function.
+The function requires specifying a unique
+database identifier argument named `db_identifier` to launch multiple databases.
+This step is necessary to connect to databases outside of the driver script.
+We will use the `db_identifier` names we specified in the application script.
+
+For the single-sharded database:
+
+.. code-block:: python
+
+  single_shard_db = exp.create_database(db_nodes=1, db_identifier="single_shard_db_identifier")
+  exp.generate(single_shard_db, overwrite=True)
+
+For the multi-sharded database:
+
+.. code-block:: python
+
+  multi_shard_db = exp.create_database(db_nodes=3, db_identifier="multi-shard-db-identifier")
+  exp.generate(multi_shard_db, overwrite=True)
+
+.. note::
+  Calling ``exp.generate()`` will create two subfolders
+  (one for each Orchestrator created in the previous step)
+  whose names are based on the db_identifier of that Orchestrator.
+  In this example, the Experiment folder is
+  named ``getting-started-multidb/``. Within this folder, two Orchestrator subfolders will
+  be created, namely ``single_shard_db_identifier/`` and ``multi_shard_db_identifier/``. It's
+  important to note that the names of the Orchestrator subfolders are determined by
+  the db_identifier value we set when instantiating an ``Orchestrator``.
```

Comment:
I think you repeat yourself here regarding the names of the subfolder being based on the db_id of the Orchestrator

Reply:
that was a mega repeat, thank u for catching!